src/wdl/workflows/RegisterEGADatasetFinalizeSubmission/RegisterEGADatasetFinalizeSubmission.wdl

```diff
@@ -0,0 +1,51 @@
+version 1.0
+
+workflow RegisterEGADatasetFinalizeSubmission {
+
+    input {
+        String submission_accession_id
+        String ega_inbox
+        String password
+        String policy_title
+        String library_strategy
+        Array [String] run_accession_ids
+        String? dataset_title
+        String? dataset_description
+    }
+
```

Comment:
would you need to call RegisterDatasetFinalizeSubmission here?

Reply:
Yes 🤦